src/composables/useTour.ts

```diff
@@ -0,0 +1,393 @@
+                target: '.account-overview .backup-warning button',
+                content: [
+                    'There is no \'forgot password\'.',
+                    'Create a backup to make sure you stay in control.',
+                ],
+                params: {
+                    placement: 'bottom',
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.account-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.account-overview .account-balance-container',
+                    '.account-overview .address-list',
+                    '.account-overview .bitcoin-account',
+                ],
+            },
+        },
+        [MobileOnboardingTourStep.MENU_ICON]: {
+            path: '/',
+            tooltip: {
+                target: '.account-overview .mobile-menu-bar > button.reset',
+                content: [
+                    'Tap on the menu icon to access your account and wallet settings.',
+                ],
+                params: {
+                    placement: 'bottom-start',
+                },
+            },
+            lifecycle: {
+                prepareDOMNextPage: async () => {
+                    await openAccountOptionsModal();
+                },
+                onMountedStep: (cbu: () => void) => {
+                    const addressButton = document
+                        .querySelector('.account-overview .mobile-menu-bar > button.reset');
+
+                    // eslint-disable-next-line no-unused-expressions
+                    addressButton?.addEventListener('click', () => cbu(), { once: true });
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.account-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.account-overview .account-balance-container',
+                    '.account-overview .address-list',
+                    '.account-overview .bitcoin-account',
+                ],
+                disabledNextStep: true,
+            },
+        },
+        [MobileOnboardingTourStep.ACCOUNT_OPTIONS]: {
+            path: '/?sidebar=true',
+            tooltip: {
+                target: '.modal .small-page',
+                content: [
+                    'Create, switch and log out of accounts.',
+                    'All security relevant actions can be found here too.',
+                ],
+                params: {
+                    placement: 'top',
+                },
+            },
+            ui: {
+                elementsWithoutInteractivity: [
+                    '.column-sidebar',
+                ],
+            },
+            lifecycle: {
+                prepareDOMPrevPage: async () => {
+                    await closeAccountOptionsModal();
+
+                    root.$router.push('/');
```

Comment:
The previous step already has its `path` defined as `'/'`, so that should be automatic. Why do you need to manually navigate here?